email/unsubscribe.js

```diff
@@ -0,0 +1,79 @@
+var fs = require("fs"),
+	jwt = require("jsonwebtoken"),
+	log = require("../lib/logger");
+
+module.exports = function (core, config) {
```

Comment:
var template = fs.readFileSync(__dirname + "/...");

// inside the file, have a word ${message}